Library/ViewModels/AddNewCardViewModel.swift

```diff
@@ -241,4 +240,9 @@ AddNewCardViewModelOutputs {
 
   public var inputs: AddNewCardViewModelInputs { return self }
   public var outputs: AddNewCardViewModelOutputs { return self }
+
+  static func publishableKey(for environment: EnvironmentType) -> String {
```

Comment:
Any reason you chose to use a `static` function here and not just a `private` function in this file?

Reply:
Ah yeah it's because otherwise I get a `'self' captured by a closure before all members were initialized` error

Reply:
Oh right but why not make this a global `private` function in the file so outside of the type definition?